appserver/web/web-core/pom.xml

```diff
@@ -169,6 +169,12 @@
             <artifactId>healthcheck-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
This had a transitive dependency on kernel, which in turn had a dependency on this.